README.md

```diff
@@ -149,6 +150,19 @@ We welcome contributions of any kind including new features, bug fixes, and docu
 
 See [MIGRATING.md](https://github.com/stripe/stripe-ios/blob/master/MIGRATING.md)
 
+## Code style
+We use [swiftlint](https://github.com/realm/SwiftLint) to enforce code style.
+
+To install it, run `brew install swiftlint`
+
+To lint your code before pushing you can run `ci_scripts/lint_modified_files.sh`
+
+You can also add this script as a pre-push hook by running `ln -s "$(pwd)/ci_scripts/lint_modified_files.sh" .git/hooks/pre-push && chmod +x .git/hooks/pre-push`
+
+Note that a substantial amount of files don't fully conform to the code style enforced by our lint rules we want to progressively improve the code style of our existing code, so if you modify a file that contains lint violations you should fix them.
+
```

Comment:
This is no longer true, I just removed it from our docs, so you should remove it here too.
```suggestion
```